MAUI/TimePicker/getting-started.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: Getting started with .NET MAUI Time Picker control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Time Picker (SfTimePicker) control and its basic features.
+platform: maui
+control: SfTimePicker
+documentation: ug
+---
+
+# Getting started with .NET MAUI Time Picker (SfTimePicker)
+This section explains how to add the Time Picker control. This section covers only the basic features needed to get started with Syncfusion Time Picker.
+
+## Creating an application using the .NET MAUI Time Picker
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfTimePicker` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Picker](https://www.nuget.org/packages/Syncfusion.Maui.Picker), and then install it.
```

Comment:
2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfTimePicker` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Picker](https://www.nuget.org/packages/Syncfusion.Maui.Picker), then install it.

Reply:
Content modified.